From 809f55ed80f106669b7b58a9457fa871a8527f64 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mikeioannina@cyanogenmod.org>
Date: Tue, 29 Mar 2016 01:30:23 +0300
Subject: [PATCH 21/28] LatinIME: Disable Hungarian spell checking

* The current dictionary is corrupt and we cannot generate a new
  one because the wordlist contains invalid entries with commas

Change-Id: I2057fd0d4ce522f66c714764a3e9bb3d8da59cb0
---
 java/res/xml/spellchecker.xml | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/java/res/xml/spellchecker.xml b/java/res/xml/spellchecker.xml
index 7a1177389..3135d09da 100644
--- a/java/res/xml/spellchecker.xml
+++ b/java/res/xml/spellchecker.xml
@@ -59,10 +59,6 @@
             android:label="@string/subtype_generic"
             android:subtypeLocale="pt_BR"
     />
-    <subtype
-            android:label="@string/subtype_generic"
-            android:subtypeLocale="hu"
-    />
     <subtype
             android:label="@string/subtype_generic"
             android:subtypeLocale="da"
-- 
2.17.1

